rllib/evaluation/collectors/simple_list_collector.py

```diff
@@ -273,20 +355,22 @@ def build(self):
                 this policy.
         """
         # Create batch from our buffers.
-        batch = SampleBatch(self.buffers)
-        assert SampleBatch.UNROLL_ID in batch.data
+        batch = SampleBatch(
```

Comment:
internal states are already time-chunked (only one internal state value every `max_seq_len` timesteps). That's why we shouldn't compare column sizes in SampleBatch (e.g. the obs column will have a different batch dim than the state_in_o one)